lib/discourse_post_event/event_validator.rb

```diff
@@ -61,6 +61,20 @@ def validate_event
         end
       end
 
+      if extracted_event[:recurrence].present?
+        valid_recurrences = %w[
+          every_month
+          every_week
+          every_two_weeks
+          every_day
+          every_weekday
+        ]
```

Comment:
Can this be extracted into a const?

Reply:
done in https://github.com/discourse/discourse-calendar/pull/226/commits/11ddb1160ad9ce57bba4994f3ce2040b2cb7e32d